client/my-sites/upgrades/domain-search/site-redirect.jsx

```diff
@@ -2,17 +2,19 @@
  * External dependencies
  */
 var page = require( 'page' ),
-	React = require( 'react' );
+	React = require( 'react' ),
+	{ connect } = require( 'react-redux' );
```

Comment:
AFAIK we don't encourage using `require()` just for consistency reasons. Refactoring the entire component to use ES6 imports can definitely be done in another PR, but we should only use `import` for any new imports we introduce.